### ISSUE TITLE
Need aria-update when changing pages in list view

### ISSUE THREAD

**Issue by seanredmond** [nypl-spacetime/maps-by-decade#90]
The aria-update works fine on the results count when you change filters, but there should be something when you change page as well.
